main.go

```diff
@@ -91,16 +91,13 @@ func main() {
 // return error in case of problems with connection initialisation
 func prepareCollectors(opts opts) (*collectors, error) {
 	var collectors = &collectors{}
-	var err error
 	if opts.PrismAPIKey != "" && opts.PrismAPIPassword != "" {
-		log.Print("[INFO] Initialising Prisma data collection")
-		collectors.prisma, err = api.NewPrisma(opts.PrismAPIKey, opts.PrismAPIPassword, opts.PrismAPIUrl)
-		if err != nil {
-			return nil, errors.Wrap(err, "can't connect to Prisma")
-		}
+		log.Printf("[INFO] Initialising Prisma data collection with API key %s", opts.PrismAPIKey)
```

Comment:
why are you logging the api key?

Reply:
It's login and would help troubleshoot the issues with authentification. Key is useless without password.